src/test/regress/expected/foreign_tables_mx.out

```diff
@@ -416,6 +416,34 @@ NOTICE:  renaming the new table to foreign_tables_schema_mx.foreign_table_local_
 (1 row)
 
 DROP FOREIGN TABLE foreign_table_local;
+-- disallow dropping table_name when foreign table is in metadata
+CREATE TABLE table_name_drop(id int);
```

Comment:
at this point, it'd be very useful to have a non postgres-fdw test.

Can we easily add a file_fdw test?

```SQL
CREATE EXTENSION file_fdw;
\copy (select i from generate_series(0,100)i) to '/tmo/test_file_fdw.data';

CREATE SERVER citustest FOREIGN DATA WRAPPER file_fdw;

CREATE FOREIGN TABLE citustest_filefdw (
 data text) SERVER citustest OPTIONS ( filename '/tmp/test_file_fdw.data');
```

and, play with it. Maybe first try checking we cannot add this as citus local table (to cover #6652), and then add table_name option to the create foreign table && make it citus local. Later, play with add/drop table_name options